csssoftvision.css

```diff
@@ -0,0 +1,44 @@
+body {
+  font: normal medium/1.4 sans-serif;
```

Comment:
Would you believe that this is the first such `font` shorthand I've seen used on an actual project (as opposed to just documentation / examples)?